app/main/reducers/profile-command-reducers.js

```diff
@@ -22,15 +23,15 @@ import { ProfileStorage } from '../../services/storage';
 
 export default async function commandHandler(storage: ProfileStorage,
                                              dispatch: any,
-                                             sessionId: number,
+                                             browserWindow: electron.BrowserWindow,
                                              command: profileActions.ProfileAction): Promise<void> {
-  console.log(`${sessionId}: Reducing ${JSON.stringify(command)}`);
+  console.log(`Reducing ${JSON.stringify(command)}`);
```

Comment:
Ditto - maybe there should be a prefix on this message like "profile-command-reducer" to make it clear where the log is coming from